include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-rcar-common.h

```diff
@@ -25,6 +25,9 @@
 /* Arbitrary number to encode non capable gpio pin */
 #define PIN_NOGPSR_START 1024U
 
+/* some of pins can have no function bits, but have to be defined as peripheral */
+#define FUNC_DUMMY 0xf
```

Comment:
I am not sure that this is the good idea in terms of future proofing. IPSR entry is 4 bit wide and theres is a chance that some Renesas SoC will use all 16 values, including 0xF. For example, Gen3 already uses functions in range 0x0-0xD.